packages/code-du-travail-frontend/__tests__/contribution.test.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import FicheContribution from "../pages/fiche-ministere-travail/[slug]";
+
+describe("<FicheContribution />", () => {
+  it("should render with external content", () => {
+    const { container } = render(
+      <FicheContribution
+        query={{
+          slug:
+            "arret-maladie-pendant-la-periode-dessai-quelles-sont-les-regles"
```

Comment:
I don't see the content in the snapshot, normal ?

Reply:
Si il y est bien